### DISCUSSION TITLE
The link to the 'View Source Code' button seems to have expired.

### DISCUSSION THREAD

**Discussion by Chen-Luan** [squidfunk/mkdocs-material#5712] [Q&A] [answered]
The link to the 'View Source Code' button seems to have expired (but the "edit" button is ok) and cannot be redirected to the correct link, whether in the MKDocs development documentation or in my documentation.

![image](https://github.com/squidfunk/mkdocs-material/assets/62653664/99854c97-6bd4-4ee8-bdcd-2a7dc25f3a2f)
![image](https://github.com/squidfunk/mkdocs-material/assets/62653664/8d592cfc-bcca-46f1-88fa-4698c7b92705)
![image](https://github.com/squidfunk/mkdocs-material/assets/62653664/ccdb1120-2fcf-4369-90e3-2cb5776734a9)

**Comment by squidfunk** [comment, marked as answer]
Works for me – I'm not able to reproduce, e.g.:
https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/docs/reference/index.md

**Reply by Chen-Luan** [reply]
Okay, I tried it and it's really normal... currently.

**Comment by gledos** [comment]
The reason why it cannot be opened is that the domain name raw.githubusercontent.com is blocked by GFW or seriously interfered.
